website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -256,19 +256,3 @@ When you set the `validate_macro_args` flag to `True`, dbt will:
 - Raise warnings if the names or types don't match
 - Validate that the `types` values follow the supported format explained in the next section
```

Comment:
since we moved the 'supported types' from this page to the `type` doc, we need to update this and add a link to the supported types i think
```suggestion
- Validate that the [`type` values follow the supported format](/reference/resource-properties/arguments#supported-types-for-macro-argument-validation).
```